src/AzureRMAssets.ts

```diff
@@ -216,7 +233,7 @@ export class FunctionMetadata {
     }
 }
 
-export interface FunctionMetadataContract {
+interface FunctionMetadataContract {
     functionSignatures?: {
         name?: string;
         expectedUsage?: string;
```

Comment:
Question: Is at least one of name or expectedUsage required?

Reply:
Not code I'm changing right now.